docs/metrics/README.md

```diff
@@ -390,18 +392,19 @@ Given a metric, once the cardinality limit is reached, any new measurement which
 cannot be independently aggregated because of the limit will be dropped or
 aggregated using the [overflow
 attribute](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#overflow-attribute)
-(if enabled). When NOT using the overflow attribute feature a warning is written
-to the [self-diagnostic log](../../src/OpenTelemetry/README.md#self-diagnostics)
-the first time an overflow is detected for a given metric.
+(if enabled). A warning is written to the [self-diagnostic
```

Comment:
the "if enabled" will no longer is applicable after this PR as this will be the default case.

Reply:
I've updated it.